### PR TITLE
custom user create, new migration create

### PR DESCRIPTION
**Pull request by Melissendra**
Could please take a look to what i did and tell me if all as you want?

### CODE REVIEW

File: morse_app/settings.py [hunk 1]
```diff
@@ -39,8 +39,39 @@
...
+    'allauth',
+    'allauth.account',
+    'allauth.socialaccount',
+
+    # local
+    'users',
 ]
 
+# django-allauth config
+SITE_ID = 1
```
Comment (TheSabby) [inline morse_app/settings.py:58]: What is this variable for?

File: morse_app/settings.py [hunk 2]
```diff
@@ -39,8 +39,39 @@
...
+ACCOUNT_EMAIL_REQUIRED = True
+ACCOUNT_USERNAME_REQUIRED = True
```
Comment (TheSabby) [inline morse_app/settings.py:60]: Do these two `_REQUIRED` variables need to be here? Seems like they are only a concern of the database models

**Review by TheSabby — APPROVED** [review]

File: morse_app/settings.py (hunk 1 above)
Comment (Melissendra) [reply morse_app/settings.py:58]: It's for Django-allauth to recognize the site's url

File: morse_app/settings.py (hunk 2 above)
Comment (Melissendra) [reply morse_app/settings.py:60]: django-allauth config needs to be in the settings of Django App

**Pull request merged**